src/main/kotlin/Stores/TeamStore.kt

```diff
@@ -0,0 +1,37 @@
+package Stores
+
+import io.ktor.http.Parameters
+import models.Team
+import models.Team.createdAt
+import models.Team.id
+import models.Team.name
+import models.Team.updateAt
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.update
+import org.joda.time.DateTime
+
+data class TeamObj (
+        var id: String,
+        var name: String,
+        var createdAt: String,
+        var updateAt: String
+)
+
+object TeamStore : BaseStore<Team>(Team) {
+    suspend fun create(params: Parameters): String? {
+        return create  {
+                it.first[name] = params["name"]!!
+        }
+
+    }
+    suspend fun getAll() : List <TeamObj>{
+        return getAll {
+            TeamObj(
+                    it[id],
```

Comment:
Querying by `id` here should be enough. Those will be unique for teams so we should only have to search by that

Reply:
this doesn't query? this just returns all the teamobjs?